WebApp/webapp/src/components/NavBar/index.module.css

```diff
@@ -0,0 +1,41 @@
+.collapsedOverlay {
+    transition: opacity 0.3s ease;
+  }
+  
+  .expandedOverlay {
+    transition: opacity 0.3s ease;
+  }
+  
+  .companyInfo {
+    display: flex;
+    align-items: center;
+    padding: 1rem;
+    gap: 1rem;
+    color: #dddddd;
+  }
+  
+  .menuSubtitle {
+    padding: 1rem;
+    color: #dddddd;
+  }
+  
+  .overlay {
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    background-color: rgba(0, 0, 0, 0.5);
+    z-index: 1;
+  }
+  
+  .grayLine {
+    height: 1px;
+    background-color: #ccc;
+    margin: 20px 0;
+  }
+  
+  .activeMenuItem {
+    background-color: #333333;
+    color: #111633;
```

Comment:
same as above:
also can you change the color to white, or whatever the standard navbar font is when its not clicked. Text is hard to see when navbar item is selected